### PR TITLE
Fix engine

### PR DESCRIPTION
**Pull request by tdfischer**

### CODE REVIEW

**Comment by tdfischer** [comment]
+r

File: test/lib.engine.js [hunk 1]
```diff
@@ -25,10 +27,21 @@ describe('lib/engine', function() {
...
       var compiler = new Compiler(config);
+      var currentDir = path.join(__dirname, '/test_contract');
+      var p = [];
+      compiler.on('file', function(event) {
+        if (event.name.indexOf(currentDir) !== 0) {
+          throw new Error('File path does not have current directory prefix: ' + event.name);
+        }
+        p.push(fileManager.storeFileWithHash(event.hash, event.data));
+      });
       var manifestHash = compiler.compileModule(path.join(__dirname, '/test_contract'));
```
Comment (wilsonianb) [inline test/lib.engine.js]: We can now just pass `currentDir` into compileModule

**Comment by wilsonianb** [comment]
Can you add `[Finishes #89035332]` to the commit message?

**Comment by tdfischer** [comment]
Done.

**Comment by wilsonianb** [comment]
LGTM +1

**Comment by codius-montecarlo** [comment]
The build is good and I'm seeing 1 +1s from the following folks:

* wilsonianb

As such, I'll attempt to merge and close this pull request.

**Pull request merged**